projects/demo/src/modules/experimental/badge-alert/examples/2/index.html

```diff
@@ -0,0 +1,4 @@
+<tui-badge-alert>10</tui-badge-alert>
```

Comment:
Shouldn't the default be 'm'?

Reply:
Idk, the new "m" size badges like old "s" size. Looks too small
Do we have any "default size rule"?